voltage/message.py

```diff
@@ -127,7 +127,7 @@ def __init__(self, data: MessagePayload, cache: CacheHandler):
         self.cache = cache
         self.id = data["_id"]
         self.created_at = ULID().decode(self.id)
-        self.content = data["content"]
+        self.content = data.get("content", "")
```

Comment:
Why are you using an empty string instead of making it Null?

Reply:
thats how dpy does it, and its better tbh

Reply:
dpy does it because discord always provides an empty string

Reply:
in addition to what olivier said, with voltage not using a None is just wrong

Reply:
oh ok, ill change it when i get home